docker-java/src/test/java/com/github/dockerjava/cmd/EventsCmdIT.java

```diff
@@ -113,6 +114,57 @@ public void testEventStreamingWithFilter() throws Exception {
         }
     }
 
+    @Test
+    public void testEventStreamingWithContainerEventTypeFilter() throws Exception {
```

Comment:
there is no need to have multiple tests for different `EventType`s, one (with `IMAGE`) is enough

Reply:
Can't hurt to have both can it?

Reply:
it can, when the test suite runs for hours instead of minutes :)

Reply:
I updated the test, so that now it iterates all EventType values and checks that, when filtered, the result only includes events of the used type